exceptioncontext.py,sha256=wD07cdqtFqebJFcA69EjiBeb5z_h161n4toD48wEiTo,2358
exceptioncontext-1.0.2.dist-info/LICENSE,sha256=gTW0XSdjEa1mV_oHlGPe72A66yFrkDs4-BFF06W6vbw,671
exceptioncontext-1.0.2.dist-info/METADATA,sha256=ul4Do1wANgxxIH60kSgrme0B_cLN8AK8fF6OmLl_J5g,2829
exceptioncontext-1.0.2.dist-info/WHEEL,sha256=h_aVn5OB2IERUjMbi2pucmR_zzWJtk303YXvhh60NJ8,110
exceptioncontext-1.0.2.dist-info/top_level.txt,sha256=fCGjpWqKY0BrfagW2z4p2s5YMthziA0lqb9GOuNqmF0,17
exceptioncontext-1.0.2.dist-info/RECORD,,
